cookbooks/crustal_model_2D.prm

```diff
@@ -104,7 +104,7 @@ subsection Mesh refinement
   set Initial global refinement          = 3
   set Refinement fraction                = 0.95
   set Strategy                           = strain rate 
-  set Coarsening fraction                = 0.05
+  set Coarsening fraction                = 0.01
```

Comment:
why are you changing this file?

Reply:
@chronictectonic - you will need to change this back to the default value, make a new commit and then squash the commits. When you are ready to do this, please ask one of us to show you how!

Reply:
This was not supposed to be part of the PR.